community/primitive-collections/src/test/java/org/neo4j/collection/primitive/hopscotch/PrimitiveLongMapTest.java

```diff
@@ -939,7 +939,7 @@ public void longLongEntryVisitorShouldSeeAllEntriesIfItDoesNotBreakOut()
     {
         // GIVEN
         PrimitiveLongLongVisitor<RuntimeException> visitor;
-        try ( PrimitiveLongLongMap map = Primitive.offHeapLongLongMap() )
+        try ( PrimitiveLongLongMap map = Primitive.longLongMap() )
```

Comment:
Maybe we should have two separate tests instead of just changing the old one?

Reply:
I have added duplicates below, it's just the git diff that looks wonky.